### ISSUE TITLE
(A11y) - Accordion component investigation

### ISSUE THREAD

**Issue by Vanals** [newscorp-ghfb/newskit#163]
Purpose

Feed a11y requirements into specification

Requirements

How the screen reader is supposed to read the component

The keyboard flow, using both TAB and SCAN (in more breakpoints if needed)

What type of information should be read out to the user on each step (each TAB press or SCAN move)

WCAG guidelines 

 

 

How this can be achieved with Semantic HTML + Aria attributes.

https://baseweb.design/components/accordion/

 

Outcome/s

Feed into/update component specification

Run team through findings 

Time box

2-3 Day

Supporting Documentation
